app/globals.css

```
@@ -0,0 +1,96 @@
+@import url('https://fonts.googleapis.com/css2?family=Geist:wght@400;500;600;700&family=Merriweather:ital,wght@0,300;0,400;0,700;1,300;1,400&display=swap');
+
+@tailwind base;
```

Biome (2.4.4)

[error] Unexpected unknown at-rule: tailwind 

(lint/suspicious/noUnknownAtRules)